src/backend/base/langflow/components/brightdata/brightdata_structured_data.py

```
@@ -0,0 +1,1108 @@
+            value=False,
+            info="Include detailed auto-detection scoring and confidence analysis in output",
+            advanced=True,
+        ),
+    ]
+
+    outputs = [
+        Output(display_name="Structured Data", name="data", method="extract_structured_data"),
+        Output(display_name="Detection Info", name="detection_info", method="get_detection_info"),
+    ]
+
+    def _get_all_datasets(self) -> Dict[str, Dict[str, Any]]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)